packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -94,6 +95,13 @@ export interface VirtualNodeBaseProps {
    * @default - No access logging
    */
   readonly accessLog?: AccessLog;
+
+  /**
+   * Default Configuration Virtual Node uses to communicate with Virtual Service
+   *
+   * @default - No Config
+   */
+  readonly backendDefaultsClientPolicy?: ClientPolicy;
```

Comment:
Same comment here as for `VirtualGateway`.

```suggestion
  readonly backendsDefaultClientPolicy?: ClientPolicy;
```